src/irr/asset/CSTLMeshWriter.cpp

```diff
@@ -304,24 +288,8 @@ void CSTLMeshWriter::writeFaceText(io::IWriteFile* file,
 		normal = core::plane3dSIMDf(vertex1, vertex2, vertex3).getNormal();
 	};
 
-	if(_params.writerFlags & IAssetWriter::EWPF_WRITE_RIGHT_HANDED)
-	{
-		if(_params.writerFlags & IAssetWriter::EWPF_MESH_IS_RIGHT_HANDED)
-		{
-			// do nothing
-		}
-		else
-			flipVectors();
-	}
-	else
-	{
-		if (!(_params.writerFlags & IAssetWriter::EWPF_MESH_IS_RIGHT_HANDED))
-		{
-			// do nothing
-		}
-		else
-			flipVectors();
-	}
+	if (!(_params.writerFlags & IAssetWriter::EWPF_MESH_IS_RIGHT_HANDED))
```

Comment:
STL is right handed too?

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/401#discussion_r347512821

Reply:
so STL AND PLY are right handed?

Reply:
they are righthanded

Reply:
cool, never knew that ;)